src/components/Explore/FavoriteButton.tsx

```diff
@@ -26,8 +27,9 @@ const FavoriteButtonContainer = styled.button`
 
 export default function FavoriteButton() {
   const theme = useTheme()
+  const [showFavorites, setShowFavorites] = useAtom(showFavoritesAtom)
   return (
-    <FavoriteButtonContainer>
+    <FavoriteButtonContainer onClick={() => setShowFavorites(!showFavorites)}>
       <FavoriteButtonContent>
```

Comment:
This should be called `FavoriteButton` and be exported. the onClick should be on the button. Then move the `FavoriteButtonContainer` into the file that uses `FavoriteButton`.

Reply:
Ah my naming schema is bad, but favoriteButtonContainer is actually a styled button -- so the onClick is on the button, I wanted to name it FavoriteButton (but that's the component name lol)

Reply:
OHH. Yeah i'm wondering if maybe we should do `StyledFavoriteButton` instead of `FavoriteButtonContainer` to make things clearer going forwards. LMK what you think.